build/WinForms.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks Condition="!$(Configuration.Contains('UniversalGL')) Or '$(Engine)' != 'UniversalGL'">net48;net6.0-windows10.0.22000.0</TargetFrameworks>
```

Comment:
Why `net6.0-windows10.0.22000.0` specifically?

Reply:
It's the latest Windows SDK version (Windows 11), making it possible to use the latest features should we want to.

Reply:
Doesn't that mean that everyone has to install Win11 SDK?

Reply:
No it has no reference to the SDK itself. Instead it adds a framework reference to Microsoft.Windows.SDK.NET.Ref for the given version.